stl/inc/xutility

```diff
@@ -4693,10 +4681,20 @@ _NODISCARD _Iter_diff_t<_FwdIt> count(
     _ExPo&& _Exec, const _FwdIt _First, const _FwdIt _Last, const _Ty& _Val) noexcept; // terminates
 #endif // _HAS_CXX17
 
-// FUNCTION TEMPLATE _Count_pr
+// FUNCTION TEMPLATE is_permutation

```

Comment:
FYI, #279 tracks a possible improvement to `is_permutation`, although it isn't necessary to implement at this time.

Reply:
I do agree that looks interesting but would prefer that be a separate change (as it'll need its own perf tests and similar justification)